### DISCUSSION TITLE
Multiple PK columns support

### DISCUSSION THREAD

**Discussion by javtau** [aminalaee/sqladmin#59] [Q&A] [answered]
Hi, this is so great, 
Are you thinking on implement the multiple pk support for foreign primary keys like flask Admin?

**Comment by aminalaee** [comment, marked as answer]
Hey @javtau ,

Thank you. Glad you enjoyed it so far.

Yes actually that's on my radar and I think I'll get to it after implementing a few core features like update view (almost done) and ajax remote queries.

Thank you for the suggestion, feel free to create the issue for tracking this or even if you feel like it, doing a PR for it 👍
